docker/README.md

```diff
@@ -1,27 +1,41 @@
 # Docker Images
+
+## Prerequisites
+You need to install [docker](https://docs.docker.com/install/) and
+[docker-compose](https://docs.docker.com/compose/install/).
```

Comment:
Only on Linux, and maybe not even required for latest version of docker?

Reply:
Sorry, what? Docker doesn't come on mac / windows. Also, this was from the old documentation.

Reply:
I'm talking about docker-compose, which was not pre-packaged with docker for Linux. Not sure if that's still the case for the latest version.

Reply:
I don't think it hurts to just explicitly state this.

Reply:
"Docker Desktop for Mac and Docker Toolbox already include Compose along with other Docker apps, so Mac users do not need to install Compose separately" & "Docker Desktop for Windows and Docker Toolbox already include Compose along with other Docker apps, so most Windows users do not need to install Compose separately". 

If you want to be explicit, say this only applies to Linux will cause less confusions. In fact that's how it was stated in quickstart: https://github.com/linkedin/datahub/blob/master/docs/quickstart.md#datahub-quickstart-guide